ebpf/tracer.go

```diff
@@ -216,7 +244,7 @@ func (t *Tracer) removeEntries(mp, tcpMp *bpflib.Map, entries []*ConnTuple) {
 		// We have to remove the PID to remove the element from the TCP Map since we don't use the pid there
 		entries[i].pid = 0
 		// We can ignore the error for this map since it will not always contain the entry
-		t.m.DeleteElement(tcpMp, unsafe.Pointer(entries[i]))
+		_ = t.m.DeleteElement(tcpMp, unsafe.Pointer(entries[i]))
```

Comment:
Whats will the explicit wildcards on the return value? gofmt?

Reply:
Not gofmt, warnings about ignoring returned errors.